workflows/targeted.nf

```diff
@@ -0,0 +1,693 @@
+include { READ_ALIGNMENT_DNA    } from '../subworkflows/local/read_alignment_dna'
+include { READ_ALIGNMENT_RNA    } from '../subworkflows/local/read_alignment_rna'
+include { READ_PROCESSING       } from '../subworkflows/local/read_processing'
+include { SAGE_APPEND           } from '../subworkflows/local/sage_append'
+include { SAGE_CALLING          } from '../subworkflows/local/sage_calling'
+
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    RUN MAIN WORKFLOW
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+// Get absolute file paths
+samplesheet = Utils.getFileObject(params.input)
+
+workflow TARGETED {
+    // Create channel for versions
+    // channel: [ versions.yml ]
+    ch_versions = Channel.empty()
+
+    // Create input channel from parsed CSV
+    // channel: [ meta ]
+    ch_inputs = Channel.fromList(inputs)
+
+    // Set up reference data, assign more human readable variables
+    PREPARE_REFERENCE(
+        run_config,
+    )
+    ref_data = PREPARE_REFERENCE.out
+    hmf_data = PREPARE_REFERENCE.out.hmf_data
+    panel_data = PREPARE_REFERENCE.out.panel_data
+
+    // Set GRIDSS config
+    gridss_config = params.gridss_config !== null ? file(params.gridss_config) : hmf_data.gridss_config
+
+    //
+    // SUBWORKFLOW: Run read alignment to generate BAMs
+    //
+    // channel: [ meta, [bam, ...], [bai, ...] ]
+    ch_align_dna_tumor_out = Channel.empty()
+    ch_align_dna_normal_out = Channel.empty()
+    ch_align_rna_tumor_out = Channel.empty()
+    if (run_config.stages.alignment) {
+
+        READ_ALIGNMENT_DNA(
+            ch_inputs,
+            ref_data.genome_fasta,
+            ref_data.genome_bwamem2_index,
+            params.max_fastq_records,
+        )
+
+        READ_ALIGNMENT_RNA(
+            ch_inputs,
+            ref_data.genome_star_index,
+        )
+
+        ch_versions = ch_versions.mix(
+            READ_ALIGNMENT_DNA.out.versions,
+            READ_ALIGNMENT_RNA.out.versions,
+        )
+
+        ch_align_dna_tumor_out = ch_align_dna_tumor_out.mix(READ_ALIGNMENT_DNA.out.dna_tumor)
+        ch_align_dna_normal_out = ch_align_dna_normal_out.mix(READ_ALIGNMENT_DNA.out.dna_normal)
+        ch_align_rna_tumor_out = ch_align_rna_tumor_out.mix(READ_ALIGNMENT_RNA.out.rna_tumor)
+
+    } else {
+
+        ch_align_dna_tumor_out = ch_inputs.map { meta -> [meta, [], []] }
+        ch_align_dna_normal_out = ch_inputs.map { meta -> [meta, [], []] }
+        ch_align_rna_tumor_out = ch_inputs.map { meta -> [meta, [], []] }
+
+    }
+
+    //
+    // SUBWORKFLOW: Run MarkDups for DNA BAMs
+    //
+    // channel: [ meta, bam, bai ]
+    ch_process_dna_tumor_out = Channel.empty()
+    ch_process_dna_normal_out = Channel.empty()
+    if (run_config.stages.markdups) {
+
+        // NOTE(SW/MC): hardcoded for initial testing purposes
+        has_umis = run_config.panel.equalsIgnoreCase('tso500')
```

Comment:
Is this still the case?

Reply:
- That's no longer relevant, removed in https://github.com/nf-core/oncoanalyser/pull/64